users/templates/base.html

```diff
@@ -76,6 +77,19 @@
             </li>
             {% endif %}
             {% if user.user_type == manager or user.user_type == admin %}
+            <li>
+                {% if request.path|startswith:custom_users_notifications %}
```

Comment:
Please don't do it this way. Use `{% with %}` to assign `if` condition result to variable and write two small `ifs` to set active class and static file path.

How about making a partial html template and reuse it for other menu items?

Reply:
Partial html template for menu items sound good, but I think that it might be another issue instead of doing this in this PR. Could you please create an issue with task @rwrzesien ?

Reply:
Added https://github.com/Code-Poets/sheetstorm/issues/430

But please still do this using `{% with %}`, as current way simply duplicates the code.

Reply:
OK nevermind, I have tried to apply this, but there were too many differences between menu items.